api/filter/http/ext_authz.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// [#not-implemented-hide:]
+// External Authorization filter calls out to an external service over the
+// gRPC Authorization API defined by api/external_auth.proto.
+// A failed check will cause this filter to return 403 Forbidden.
+message ExtAuthz {
+
+  // The name of the upstream authorization cluster.
+  string cluster_name = 1 [(validate.rules).string.min_bytes = 1];
```

Comment:
If this is for gRPC, I think gRPC refers the server as "service", not as "cluster". While you can run a cluster for a service, but it often is not the case. You can run one service across clusters or a cluster runs multiple service.

Reply:
Envoy refers to this as a cluster. Please leave it. This is Envoy related configuration that has no meaning to someone outside of Envoy. Please use Envoy terminology.